layouts/shortcodes/resource_link.html

```diff
@@ -0,0 +1,3 @@
+{{ $resource_name := .Get 0 }}
+
+{{ (.Page.Resources.GetMatch $resource_name).RelPermalink }}
```

Comment:
Shiny, does this mean that we get build-time validation of these links?

Reply:
It does, though the error message is _nasty_:
```
INFO 2018/08/04 09:43:08 Using config file: /home/peter/sb/docs/config.yml
Building sites … INFO 2018/08/04 09:43:08 syncing static files to /home/peter/sb/docs/public/
INFO 2018/08/04 09:43:08 found taxonomies: map[string]string{}
panic: reflect: Method on nil interface value [recovered]
	panic: reflect: Method on nil interface value

goroutine 148 [running]:
text/template.errRecover(0xc420f0d7f0)
	/usr/local/go/src/text/template/exec.go:143 +0x1ba
panic(0xd761a0, 0x1072ec0)
	/usr/local/go/src/runtime/panic.go:502 +0x229
reflect.Value.Method(0xe031c0, 0xc4205dff68, 0x94, 0x5, 0xc, 0x0, 0x0)
	/usr/local/go/src/reflect/value.go:1151 +0x14f
reflect.Value.MethodByName(0xe031c0, 0xc4205dff68, 0x94, 0xc4206914ed, 0xc, 0x94, 0x1, 0x10841c0)
	/usr/local/go/src/reflect/value.go:1186 +0x11b
<trimmed>
```

Reply:
This is probably preferable to no error message, but I'm adding it here for reference.